### PR TITLE
Get Job tests and GetJobChunk

### PR DESCRIPTION
**Pull request by GtJosh**

### CODE REVIEW

File: ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    }
+
+    @AfterClass
+    public static void teardown() throws IOException, SignatureException {
+        deleteAllContents(client, BUCKET_NAME);
+        TempStorageUtil.teardown(TEST_ENV_NAME, envStorageIds, client);
+        client.close();
+    }
+
+    private static void waitForObjectToBeInCache(final UUID jobId) throws Exception {
```
Comment (rpmoore) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java]: Doesn't this function exist in another test file? or one similiar?

File: ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        client.putObject(new PutObjectRequest(BUCKET_NAME, book1, book1Channel, jobId, 0, Files.size(objPath1)));
+        waitForObjectToBeInCache(jobId);
+    }
+
+    @SuppressWarnings("deprecation")
+    @Test
+    public void nakedS3Get() throws IOException, SignatureException, XmlProcessingException,
+            URISyntaxException, InterruptedException {
+
+        final WritableByteChannel writeChannel = Channels.newChannel(new OutputStream() {
```
Comment (rpmoore) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java]: We have a NullChannel implementation that does what you want here.  Use that instead.

File: ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java (hunk 1 above)
Comment (GtJosh) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java]: Yep.  Can I assume they'll always be together?

File: ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    }
+
+    @Ignore("Ignore pending implementation of withName option")
+    @Test
+    public void createReadJobWithNameOption() throws IOException, SignatureException, XmlProcessingException,
+            URISyntaxException, InterruptedException {
+
+        //TODO: REMOVE COMMENT BELOW AND VERIFY SYNTAX FOR JOB NAME
+        final Ds3ClientHelpers.Job readJob = HELPERS.startReadJob(BUCKET_NAME, Lists.newArrayList(
+                new Ds3Object("beowulf.txt", 10))/*, ReadJobOptions.create().withName("test_job")*/);
```
Comment (rpmoore) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java]: If you'd like you can go ahead and add the name property to the job options.

File: ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java (hunk 1 above)
Comment (rpmoore) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java]: What do you mean together?  in the same module?

Comment (GtJosh) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java]: Yeah--is it ok to call the function from the other file, or should I move the function to Test Helpers or something like that?

Comment (rpmoore) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java]: Yes, please move it to the test helper function

File: ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java (hunk 2 above)
Comment (GtJosh) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java]: Got it.

File: ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java (hunk 1 above)
Comment (GtJosh) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java]: Ok

File: ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java (hunk 3 above)
Comment (GtJosh) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java]: done

**Comment by rpmoore** [comment]
This will also need to get merged with 3_2_autogen

**Pull request merged**